### PR TITLE
Update Readme.txt

### PR DESCRIPTION
**Pull request by scotthibbs**
Proposed readme.txt for 1.82

### CODE REVIEW

**Comment by scotthibbs** [comment]
OOPs.. 

Hey, who helped code the cut numbers? 

### Version 1.82 (Dec 2022)
  - Added ARRL DX Contest (Coded by W7SST)
  - Add `Help | First Time Setup` (Written by KD4SIR)
  - Add cut numbers (e.g. 5NN = 599) in exchanges received (Coder??)
  - others...

**Comment by scotthibbs** [comment]
Once I have this the readme and the release draft will be ready for review.

File: Readme.txt
Comment (scotthibbs) [inline Readme.txt:289]: oops.. I need to ask who helped code the received cut numbers.

**Review by scotthibbs — COMMENTED** [review]
I need to finish this once I know who helped with received cut numbers. 

Did we miss any other additions in this version? I know ARRL DX was a biggie. But anything else done?

File: Readme.txt
Comment (w7sst) [inline Readme.txt:5]: Hmmm... Not sure of the name here. This will be a hard pattern to follow each release, especially when we have multiple contests being added.
- Initially I was thinking this could be a theme-oriented name, perhaps indicating the goal of the current milestone/release. In this case for v1.82, it is the User Group Announcement Release. So perhaps this line should be `Version 1.82 - User Group Announcement`.
- On the other hand, perhaps we should simply this to be similar to MR splash screen. Perhaps:
```
                                             MORSE RUNNER
                                          Contest Simulator
                                               freeware

                                             Version 1.82
```

File: Readme.txt
Comment (w7sst) [inline Readme.txt:11]: Should this (2023) wait till next year, assuming we publish the release this month?

File: Readme.txt
Comment (w7sst) [inline Readme.txt:298]: I'm not sure I like the feel of adding the `CE ` prefix. it seems confusing. just a thought.

File: Readme.txt
Comment (w7sst) [inline Readme.txt:281]: Does this work for all contests?

File: Readme.txt
Comment (w7sst) [inline Readme.txt:43]: *Exchanges - spelling

File: Readme.txt
Comment (w7sst) [inline Readme.txt:45]: This seems like a lot of extra detail to have this early in the document. Perhaps we should move this to the end as an Appendix?

File: Readme.txt
Comment (w7sst) [inline Readme.txt:6]: Not sure this line adds value long term.

File: Readme.txt
Comment (w7sst) [inline Readme.txt:19]: This is a minor point, but I see your editor is adding whitespace at the end of some lines.

**Review by w7sst — CHANGES_REQUESTED** [review]
Hi Scott,
Thanks for the updates to `readme.txt`. You can see my comments in the review. Let me know what you think. Also let me know if you need any help making these changes since you are editing in the github interface.

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt:5]: I like the name here.. My reasoning is that this is for the individual users. They are going to refer to this name instead of the version number when talking to others. The majority of the individual users will be unaware of what the User Group Announcement is pertaining to. When we have multiple contests or features we can simplify the name to be more generic like the Spring 23 version or the Five More Contests version. This also "sells" why they need to upgrade more than a number. When I told KD0EE we had a ARRL DX version coming out he was very excited, not as excited if I said 1.82. It's a cheerleading thing.. lol 

But honestly, I'm easy as Sunday morning on all these changes so let me know what you decide.

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt:11]: I pondered this too, I'll remove it.

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt:298]: I noticed all the other entries just had the number and not the leading "version" word. Which then I thought was cool because it gave us the ability to distinguish our versions from others. Then I thought, Community Edition was too long so I added CE so that there was a reason for the "version" word.  I guess we are the only ones that need to keep track. 

I'll remove it, but I really do like the version word so we know the community edition work from the others.

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt:281]: oh.. good catch cuz we got extra contests now! I'll clarify this.

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt:45]: Agreed. I tried putting it later but struggled with it. Even debated even adding it. But it's good info here until we get it in the help menu. 

Appendix solves this for me. :)

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt:6]: long term -- like in the future readme where it changes to "The fourteenth release of the Morse Runner Community Edition"?  

I'm viewing the readme as a help document as well as a sales pitch for MRCE awesomeness. :) I thought it was a neat tidbit to be proud of.. Our first year and three versions. We did great!!! Field Day alone was killer but now ARRL DX!!! My job is to brag on you guys. I don't have to be humble. :)

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt:19]: I'll have to watch for this. I'm using the editor within the GitHub webpage. Thanks for the tip!

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt:281]: Changed wording to add HST

**Comment by w7sst** [comment]
> I need to finish this once I know who helped with received cut numbers.
> 
> Did we miss any other additions in this version? I know ARRL DX was a biggie. But anything else done?

We usually run the `Generate Release Notes` step in the Release 1.82 Draft. This step will extract the detail from github and update the Release 1.82 section. We then edit this generated `What's changed` section and copy into this `readme.txt` file.

File: Readme.txt
Comment (w7sst) [reply Readme.txt:5]: You convinced me. I see your point and agree. Having this name does make sense to an end user community. Perhaps the name I'm looking at is more of a milestone or goal name. I'm good with your change.

File: Readme.txt
Comment (w7sst) [reply Readme.txt:6]: okay. These are release notes, not marketing or advertising materials. I good with this.

File: Readme.txt
Comment (w7sst) [reply Readme.txt]: w7sst implemented the cut number changes.

File: Readme.txt
Comment (w7sst) [reply Readme.txt]: Hopefully we will roll to Version 2.0 soon, then 2.0+ can represent to CE versions into the future. Perhaps we should get to Version 2.0 sooner than later. I was hoping to wait until N1MM integration was achieved before considering a big version number change.

I wonder if we should be considering this release as 2.0? Perhaps the 1.8x series was based on the prototype and now that we are ready to announce this things, we can call it 2.0? I don't know if we are ready or not.

Comment (w7sst) [reply Readme.txt]: Perhaps waiting for a future release before rolling to 2.0, not now.

File: Readme.txt
Comment (w7sst) [reply Readme.txt]: Or we can put this as a separate document or into a future `Help` file and not the release notes.

Comment (scotthibbs) [reply Readme.txt]: Yes that is precisely it. Our readme is currently the help file. There is an issue assigned to me to write a separate help file. It should be ready for the next release. 

So this readme will point to a help file in the program for those looking for it in the future.

**Review by scotthibbs — COMMENTED** [review]
I think all the changes have been resolved with the document. 

I'm go for launch for 1.82.

**Comment by scotthibbs** [comment]
> We usually run the `Generate Release Notes` step in the Release 1.82 Draft. This step will extract the detail from github and update the Release 1.82 section. We then edit this generated `What's changed` section and copy into this `readme.txt` file.

I noticed that the generate was already ran for the draft but I didn't see an issue resolved for the cut numbers. (I can't run the generate twice to know) I concluded that it was edited out, under a different named issue, or because the issue for cut numbers (for sending) wasn't finished yet. Since it was a significant change for the user, I did want to highlight that this was done. 

Also, I was surprised by the hiding resolved. I had to learn to not resolve until the changes were made, then the resolved hiding is very intuitive and helpful to know what is done. Especially for items where I needed to ask a question and it was answered. I did resolve someone else's proposed edit when that was accepted and it felt natural to do so so that I knew the change was already made. Again helping me know what was needed. 

Should there be a separate discussion or issue for this? Because I'm go for launch for 1.82.. 

Don't forget to check make a discussion for 1.82 before publishing. This setting is never saved in the draft.

File: Readme.txt
Comment (w7sst) [reply Readme.txt]: I think I understand now - yes, this `readme.txt` also serves as the Help file. The future readme file will probably be more than a simple text file, perhaps using Windows Help file format, or even a PDF. It would be nice to find a tool for building a help file. We also want a Help file design that will lead to future localization. We want to pick something that be integrated into MRCE to allow F1 key to work (when not in Run Mode). So much to consider.

Comment (scotthibbs) [reply Readme.txt]: I quoted the above in [Help File needs to be written](https://github.com/w7sst/MorseRunner/issues/67#top)
#67 for reference. 

Should I resolve?

**Comment by w7sst** [comment]
@scotthibbs - I regenerated the draft Release 1.82 page. It now contains the full set of changes since 1.81. You may be able to extract some other noteworthy high-points to include in this `readme.txt` and also summarize in the summary section of the Release Notes page (the section 1.82 summary). this summary is usually similar to the `readme.txt` file summary.

**Comment by w7sst** [comment]
Scott, I'm good with your changes. I'd like to see feedback from one or two others before we merge. We have all next week as we wait to hear back from ARRL. Thank you for your work on the release notes and improving the `readme.txt` file.

File: Readme.txt
Comment (w7sst) [reply Readme.txt]: I marked resolved for you.

**Review by scotthibbs — COMMENTED** [review]
I updated this from the what's changed information. 

  - Added ARRL DX Contest (Coded by W7SST)
  - CWOPS Contest fixes (Coded by W7SST)
  - Rename the F2 button to exchange (Coded by tekenny)
  - Add `Help | First Time Setup` (Written by KD4SIR & W7SST)
  - Add cut numbers (e.g. 5NN = 599) in exchanges received (Coded by W7SST)
  - bug fixes

**Review by ct7aup — APPROVED** [review]

**Pull request merged**
